stylesheets/_session_theme.scss

```diff
@@ -52,26 +68,17 @@
 
       a {
         text-decoration: underline;
-        color: var(--color-received-message-text);
+        color: inherit;
       }
     }
   }
+  color: var(--color-sent-message-text);
```

Comment:
I think this should be at the top of the class with the other properties and not in between two child classes. So move to line 31 for example.

Reply:
:+1: